### PR TITLE
In reference to previous commit CONDA

### PR DESCRIPTION
**Pull request by shakeebmurtaza**
For starting with conda environment

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - openssl=1.1.1k=h2bbff1b_0
+  - pip=21.1.2=pyhd8ed1ab_0
+  - python=3.9.5=h6244533_3
+  - setuptools=52.0.0=py39haa95532_0
+  - sqlite=3.35.5=h8ffe710_0
+  - tzdata=2021a=he74cb21_0
+  - vc=14.2=h21ff451_1
+  - vs2015_runtime=14.28.29325=h5e1d092_4
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - wincertstore=0.2=py39h2bbff1b_0
```
Comment (keflavich) [inline environment.yml:20]: these look over-specific - do we need all these specific versions?  I suspect not.

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: img_registration
+channels:
+  - defaults
+  - alex-marty
+  - conda-forge
+  - anaconda
+  - pytorch
```
Comment (keflavich) [inline environment.yml:7]: this shouldn't depend on alex-marty or pytorch

**Review by shakeebmurtaza — COMMENTED** [review]
changed as per suggestions

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: img_registration
```
Comment (keflavich) [inline environment.yml:1]:
```suggestion
name: image_registration
```

**Pull request merged**